### PR TITLE
Display error message if user provides the wrong username or password

### LINKED ISSUE #336: Display error message if user provides the wrong username/password

### PR DESCRIPTION
**Pull request by Kirill250402**
Fixes #336

### CODE REVIEW

File: app/src/main/res/values-en-rGB/strings.xml [hunk 1]
```diff
@@ -6,4 +6,5 @@
         <item>lbs</item>
         <item>tons</item>
     </string-array>
+    <string name="invalid_email_or_password">Invalid email or password</string>
```
Comment (lebedynskyi) [inline app/src/main/res/values-en-rGB/strings.xml:9]: We do not need this strings here

File: app/src/main/res/values-en-rUS/strings.xml [hunk 2]
```diff
@@ -6,4 +6,5 @@
         <item>kg</item>
         <item>tonnes</item>
     </string-array>
+    <string name="invalid_email_or_password">Invalid email or password</string>
```
Comment (lebedynskyi) [inline app/src/main/res/values-en-rUS/strings.xml:9]: We do not need this strings here

**Review by lebedynskyi — CHANGES_REQUESTED** [review]
Please remove extra strings

File: app/src/main/res/values-en-rUS/strings.xml (hunk 2 above)
Comment (Kirill250402) [reply app/src/main/res/values-en-rUS/strings.xml:9]: That was auto create.

**Review by lebedynskyi — APPROVED** [review]
Approve

**Pull request merged**